blocks/init/src/Blocks/components/image/components/image-editor.js

```diff
@@ -56,7 +57,12 @@ export const ImageEditor = (attributes) => {
 								{(!imageUsePlaceholder) ?
 									<MediaPlaceholder
 										icon="format-image"
-										onSelect={(value) => setAttributes({ [`${componentName}Url`]: value.url })}
+										onSelect={(value) => {
+											setAttributes({
+												[`${componentName}Url`]: value.url,
+												[`${componentName}Alt`]: value.alt
```

Comment:
alt is set in the options so no need to set it here

Reply:
This actually has to be set, otherwise the alt won't be shown. Not 100% sure why. I've removed it from the options and it works, but when I remove it from the editor, the alt text is not displayed in the editor or the options textarea 🤷🏼‍♂️